app/assets/stylesheets/base.scss

```diff
@@ -5,6 +5,7 @@
   --link-hover-decoration-line: underline;
   --link-hover-decoration-style: solid;
   --link-text-decoration-color: gray;
+  scroll-behavior: auto;
```

Comment:
Can you add a comment here, or improve the commit message explaining the issues this is solving?